src/templates/season/list.pug

```diff
@@ -27,6 +28,8 @@ block content
               td= season.number
```

Comment:
I believe that's just pug's syntax. I could be wrong, but it's at least what's used in all the pug docs so I don't see any good reason not to just use their style.